Jenkinsfile

```diff
@@ -7,5 +7,54 @@
 // Test plugin compatibility to latest Jenkins LTS
 // Allow failing tests to retry execution
 buildPlugin(jenkinsVersions: [null, '2.60.1'],
-            findbugs: [run:true, archive:true, unstableTotalAll: '0'],
-            failFast: false)
+        findbugs: [run: true, archive: true, unstableTotalAll: '0'],
+        failFast: false)
+
+def branches = [:]
+
+branches["ATH"] = {
+    node("docker && highmem") {
+        deleteDir()
+        stage("ATH: Checkout") {
+            dir("git") {
+                checkout scm
+                sh "mvn clean package -DskipTests"
+                dir("target") {
+                    stash name: "localPlugins", includes: "*.hpi"
+                }
+                sh "mvn clean"
+            }
+        }
+        def metadataPath = pwd() + "/git/essentials.yml"
+        stage("Run ATH") {
+            dir("ath") {
+                runATH metadataFile: metadataPath
+                deleteDir()
+            }
+        }
+    }
+}
+branches["PCT"] = {
+    node("docker && highmem") {
+        deleteDir()
+        env.RUN_PCT_LOCAL_PLUGIN_SOURCES_STASH_NAME = "localPluginsPCT"
+        stage("PCT: Checkout") {
+            dir("git") {
+                checkout scm
+            }
+            stash name: "localPluginsPCT", useDefaultExcludes: false
+
+        }
+        def metadataPath = pwd() + "/git/essentials.yml"
+
+        stage("Run PCT") {
+            dir("pct") {
+                runPCT metadataFile: metadataPath,
+                        javaOptions: ["-Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn"]
```

Comment:
I think we should set it by default in `runPCT()`. Just to KISS

Reply:
Done, runATH and runPCT apply that property by default once [pipeline-library#48](https://github.com/jenkins-infra/pipeline-library/pull/48/) is merged.

Note that if you are applying custom java opts in PCT you will still need to use it